addons/captives/stringtable.xml

```diff
@@ -204,6 +209,7 @@
             <Czech>Mohou hráči spoutat jednotky na své straně</Czech>
             <German>Können Spieler eigene Einheiten fesseln</German>
```

Comment:
even though not topic of this translation i would change "Kann Teamkollegen fesseln" in "Kann eigene Fraktion fesseln"